go/parser.go

```diff
@@ -220,6 +220,12 @@ func (a *andExpr) ToString() string {
 	return fmt.Sprintf("( %s and %s )", a.leftExpr.ToString(), a.rightExpr.ToString())
 }
 
+func IsBinaryOperator(e Evaluatable) bool {
```

Comment:
Go exports functions that start with a capital letter. Changing it to lowercase will solve that.

Reply:
Thanks for the info.
DONE.